src/Congo.WebApi/Startup.cs

```diff
@@ -24,8 +28,27 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddAppServices(Configuration);
-
             services.AddControllers();
+
+            services.AddCors(options =>
+            {
+                options.AddPolicy(_CORS_DEV, policy =>
+                            policy.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader());
+
+                options.AddPolicy(_CORS_PROD, policy =>
+                {
+                    //TODO: Multiple Origins from env vars
+                    string ALLOWED_ORIGIN = Environment.GetEnvironmentVariable("ALLOWED_ORIGIN");
+                    if (ALLOWED_ORIGIN != null)
```

Comment:
If we are going to check the value, we might aswell go a little further and make sure its not an obvious incorrect value either.
```suggestion
                    if (string.IsNullOrWhiteSpace(ALLOWED_ORIGIN))
```

Reply:
TIL. Thanks

Reply:
Oops, looks like I accidentally inverted the if. Be aware of that :)